__test__/lambda.test.js

```diff
@@ -1,10 +1,10 @@
 import { handler } from "../lambda.js";
 import { setupTest, cleanupTest } from "./test_helpers";
 
-beforeAll(
-  async () => await setupTest(),
-  global.DATABASE_SETUP_TEARDOWN_TIMEOUT
-);
+beforeAll(async () => {
+  await cleanupTest();
```

Comment:
Learning question, not a critique: I'm guessing this is now needed because we're actually using a db and not a fake db, is that right?

Reply:
my guess is we load db info a little 'earlier' in the spoke app loading now (probably from knex migrations changes) -- and so whereas before this would run/load cleanly without a db cleanup, it now needs to load correctly.